### ISSUE TITLE
Exclude site relation multipolygon members from "Missing tag or role"

### ISSUE THREAD

**Issue by sammyhawkrad** [osm-fr/osmose-backend#2343]
According to [Relation:site](https://wiki.openstreetmap.org/w/index.php?title=Relation:site&uselang=en-GB), the members are added without having to specify a role.

However, it appears that this does not apply to multipolygon members in https://github.com/osm-fr/osmose-backend/blob/master/analysers/analyser_osmosis_useless.py#L137 as it flags multipolygon members of site relations for missing tag or role.

Examples:
1. https://osmose.openstreetmap.fr/map/#level=3&item=1140&zoom=17&lat=7.4439077&lon=-2.5741419&level=3&tags=&fixable=&issue_uuid=895e4aa6-0b80-7b48-b578-8b491c08a87a
2. https://osmose.openstreetmap.fr/map/#level=3&item=1140&zoom=17&lat=7.443392&lon=-2.5742938&level=3&tags=&fixable=&issue_uuid=ae52f00d-3a70-d135-1dab-8137f27bb212
3. https://osmose.openstreetmap.fr/map/#level=3&item=1140&zoom=17&lat=7.4444908&lon=-2.5744951&level=3&tags=&fixable=&issue_uuid=a98d55f3-3626-1194-eb50-f20af1e96f42

**Comment by Marc-marc-marc** [comment]
looking at the 1st exemple, it's a wrong usage of a type=site relation : a site relation is not a collection of all objets (relation is not a collection) in an area related to one tag (operator, owner or whatelse) but for objets unable to be describe with a (closed) area
from wiki `A site [relation](https://wiki.openstreetmap.org/wiki/Relation) is used to group several objects which together have a single identity as a whole but cannot be accurately described by other data types`
here the "site" is described with the closed way

**Comment by sammyhawkrad** [comment]
I totally agree that the site relation is not correctly used in the examples I have shared. I have deleted a few of them with similar use recently. But the issue here is not really about "what" is mapped. 

The real issue is that when a multipolygon is added to a site relation it gets flagged as not having a tag or role. The issue will still arise in cases where the site relation is correctly used.

**Comment by Famlam** [comment]
Item 1140, class 2
Can you give an example of such a valid case?

I can't think of many ways (no pun intended) how an _untagged_, _outer_ way of a multipolygon is part of a site relation, but not the multipolygon (which contains the tags) itself.

**Comment by Marc-marc-marc** [comment]
wind site turbine with mixed node/way with agricultural use of the space between the turbines
